### DISCUSSION TITLE
GU16

### DISCUSSION THREAD

**Discussion by vladololov** [Taludas/WholesomeHaciendaHaul#6] [General]
Will there be a GU16 update? Because game crashes with the mode on.

**Comment by Taludas** [comment]
Yes, a patch for GU 16 is already in the making. Please a little more patience.
 
 
  On Fri, 9 Dec 2022 at 11:16, ***@***.***> wrote:   


Will there be a GU16 update? Because game crashes with the mode on.

**Reply by vladololov** [reply]
Thank you for your great work!
